menu/category.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<script>
+
+    function aa () {
+            $('.category-container').hide()
+            $('.item-description').show();
+    }
+
+
+</script>
+<body>
+
+
+<div class="category-container">
+    <div class="item-category" id="category_1" onclick="aa()">
+        <img src="/menu/img1.jpg" width="200px" height="100px"> </br>
+        <span>Chicken1</span>
+    </div>
+
+    <div class="item-category">
+        <img src="/menu/img1.jpg" width="200px" height="100px"> </br>
+        <span>Chicken2</span>
+    </div>
+
+    <div class="item-category">
+        <img src="/menu/img1.jpg" width="200px" height="100px"> </br>
+        <span>Chicken3</span>
+    </div>
+
+
+    <div class="item-category">
+        <img src="/menu/img1.jpg" width="200px" height="100px"> </br>
+        <span>Chicken4</span>
+    </div>
+
+
+    <div class="item-category">
+        <img src="/menu/img1.jpg" width="200px" height="100px"> </br>
+        <span>Chicken5</span>
+    </div>
+
+
+    <div class="item-category">
+        <img src="/menu/img1.jpg" width="200px" height="100px"> </br>
+        <span>Chicken6</span>
+    </div>
```

Comment:
try to get this array of categories from js file so you will not need to copy paste code multiple times